source/initial-dns-seedlist-discovery/tests/txt-record-with-overridden-ssl-option.json

```diff
@@ -0,0 +1,22 @@
+{
+  "uri": "mongodb+srv://test5.test.build.10gen.cc/?ssl=false",
+  "seeds": [
+    "localhost.test.build.10gen.cc:27017"
+  ],
+  "hosts": [
+    "localhost:27017",
+    "localhost:27018",
+    "localhost:27019"
+  ],
+  "options": [
+    {
+      "authSource": "otherDB"
```

Comment:
authSource is not specified in the uri, so it should be "thisDB" as specified in the TXT record for test5

Reply:
Yes, I just noticed. Fixing.